conf/application.local-base.conf

```diff
@@ -0,0 +1,38 @@
+# This base configuration file that contains values used in local development, regardless of which backend
+# is being used (local or integration).
+
+# https://www.playframework.com/documentation/latest/Configuration
+play.http.port=9000
+
+auth.callback="http://localhost:9000/callback"
+auth.secret=${AUTH_SECRET}
+
+play.cache.redis {
```

Comment:
Everything below this line has been copied straight from application.conf, because it's the same in both cases (fully local dev and integration backend).